src/Decimal.php

```diff
@@ -149,7 +142,7 @@ protected function normalizeValue(string $value): string
      *
      * @return static
      */
-    public static function create($value, ?int $scale = null)
+    public static function create($value, ?int $scale = null): self
```

Comment:
self would only be valid for PHP 8+, right?
Since this lib is still also 7.4+, we better omit this one - see also docblock

Reply:
No, I have tested this with PHP 7.3 and above

Reply:
The same reason than above though:
For these changes we need to start a new "major" branch
As the current signatures for public API would be a breaking change for all consumers that extend this class.

Reply:
Oh okay, I forgot that the class can be extend.